src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -145,7 +145,9 @@ export const AddressModal: React.FC<Props> = ({
                 onChange={formik.handleChange}
                 onBlur={formik.handleBlur}
                 error={formik.touched.phoneNumber && formik.errors.phoneNumber}
-                value={formik.values?.phoneNumber}
+                value={
+                  formik.values?.phoneNumber ? formik.values?.phoneNumber : ""
```

Comment:
non-blocking, minor: Is the purpose of this to return an empty string instead of undefined? Maybe this could be easier to read?:
```
value={formik.values?.phoneNumber || ""}
```